shared/settings/feedback/container.desktop.tsx

```diff
@@ -1,12 +1,11 @@
-// @flow
 import Feedback from './index'
 import {namedConnect} from '../../util/container'
 import * as RouteTreeGen from '../../actions/route-tree-gen'
 import * as SettingsGen from '../../actions/settings-gen'
 import {anyWaiting} from '../../constants/waiting'
 import * as Constants from '../../constants/settings'
 
-type OwnProps = {||}
+type OwnProps = {}
 
 const mapStateToProps = state => ({
```

Comment:
Oh, one more thing, when you remove the generics below, I think you'll have to type this function. i.e. `const mapStateToProps = (state: TypedState, ownProps: {}) ...`